ruby-training/trainees/alan/test_katas/03/JadenCase/spec/jaden_case_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rspec/autorun"
+require "../jaden_case.rb"
+
+describe JadenCase do
+
+    describe ".to_word_array" do
+        it "should return an array of each word in the sentence" do
+            expect(JadenCase.to_array_of_words("Hello, there.")).to eq(["Hello,","there."])
+        end
+    end
+
+    describe ".capitalize_words" do
+        it "should return an array of capitalized words" do
+            expect(JadenCase.capitalize_words!("Hello, there.")).to eq(["Hello,","There."])
+        end
+    end
+
+    describe ".make_it_in_jaden_case" do
+        it "should return the sentence with the jade case" do
```

Comment:
it "should return the sentence **in** Jaden Case" may be more grammatically correct.